composer.json

```diff
@@ -62,6 +62,8 @@
             "Phpactor\\Extension\\LanguageServer\\Tests\\": "tests/LanguageServer/",
             "Phpactor\\Extension\\LanguageServerCompletion\\Tests\\": "tests/LanguageServerCompletion/",
             "Phpactor\\Extension\\LanguageServerReferenceFinder\\Tests\\": "tests/LanguageServerReferenceFinder/",
+            "Phpactor\\Extension\\LanguageServerRename\\Tests\\": "tests/LanguageServerRename/",
+            "Phpactor\\Extension\\LanguageServerSelectionRange\\Tests\\": "tests/LanguageServerSelectionRange/",
```

Comment:
Not really, otherwise "Fix namespace..." didn't work properly.

Reply:
I needed that and composer install (supposedly to update the autoloader) for it to work. The other namespace I added I think you've missed because tests have weird namespace (note the missing starting letter of Selection):
```php
namespace Phpactor\Extension\LanguageServer\Tests\electionRange\Unit\Model;
```